src/qibolab/instruments/qm.py

```diff
@@ -562,7 +562,7 @@ def play_pulses(qmsequence, relaxation_time=0):
                 save(qmpulse.shot, qmpulse.shots)
 
     @staticmethod
-    def fetch_results(result, ro_pulses):
+    def fetch_results(result, ro_pulses, average):
```

Comment:
This will fail because the `average` argument is not given in `play` and `sweep` where this function is called.